lib/vagrant-cloudstack/config.rb

```diff
@@ -417,6 +422,9 @@ def finalize!
         # private ip is nil by default
         @private_ip_address     = nil if @private_ip_address == UNSET_VALUE
 
+        # expunge on destroy is nil by default
+        @expunge_on_destroy     = nil if @expunge_on_destroy == UNSET_VALUE
```

Comment:
I thought about that. But I'm not completely clear on what will happen if I pass this option to a cloudstack API that doesn't support the expunge option. Will it silently ignore adding extra parameters to the call?